templates/repo/issue/view_content/comments.tmpl

```diff
@@ -838,6 +838,19 @@
 					{{end}}
 				</span>
 			</div>
+<<<<<<< HEAD
```

Comment:
?

Reply:
that looks like a merge artifact, from one of the previous merge conflict resolutions.